content/docs/observability/deployment/online.md

```diff
@@ -30,9 +30,8 @@ The Container Storage Modules (CSM) for Observability installer bootstraps Helm
 
 If the Authorization module is enabled for the CSI drivers installed in the same Kubernetes cluster, the installer will perform the current steps to enable CSM for Observability to use the same Authorization instance:
 - Verifies the `karavictl` binary is available.
-- Verifies the appropriate Secret exists in the CSI driver namespace.
-- Queries the CSI driver environment to get references to the Authorization module sidecar-proxy Docker image and URL of the proxy server.
-- Updates the CSM for Observability deployment to use the existing Authorization instance.
+- Verifies the appropriate Secrets and ConfigMap exist in the CSI driver namespace.
+- Enabled during the initial installation of CSM for Observability or updates the CSM for Observability deployment to use the existing Authorization instance.
```

Comment:
Perhaps: "Updates the CSM for Observability deployment to use the existing Authorization instance if not already enabled during the initial installation of CSM for Observability"?

Reply:
fixed